source/includes/fact-older-version-limitations.rst

```diff
@@ -0,0 +1,40 @@
+- Writes that produce DDL events cannot occur on the source cluster during the 
+  migration. The following events cannot occur: 
+  
+  - create
+  - drop
+  - dropDatabase
+  - createIndexes
+  - dropIndexes
+  - collModrefineCollectionShardKey
+  - rename
+  - reshardCollection
+  - shardCollection
+
+  This includes operations that may create new collections such as 
+  :dbcommand:`mapReduce`, :pipeline:`$out`, and :pipeline:`$merge`. This also 
+  includes collections created implicitly from inserts. Only writes that produce 
+  CRUD events can occur during the migration.
+
+  .. note:: 
+   
+     Writes that produce DDL events on source collections outside of the 
+     :ref:`namespace filter <c2c-filtered-sync>` are allowed.
+
+- :ref:`GeoHaystack indexes <index-geohaystack-index>` are not supported.
```

Comment:
[Question]

Do we need to mention this? These have been removed since 5.0 and we don't have any docs on them anymore.

I think if we're keeping this, we want `geoHaystack` to be in camel case with a lower-case `g`.

Reply:
Yup I agree with Ali. Also I think we still have some docs on them: https://www.mongodb.com/docs/manual/core/indexes/index-types/geospatial/geohaystack/

Reply:
yup that works!